drivers/display/Kconfig.ssd1306

```diff
@@ -5,7 +5,7 @@
 
 menuconfig SSD1306
 	bool "SSD1306 display driver"
-	depends on I2C
+	depends on (I2C && HAS_DTS_I2C) || SPI
```

Comment:
Remove HAS_DTS_I2C it is no longer needed, https://github.com/zephyrproject-rtos/zephyr/pull/28517

Reply:
done